packages/edit-site/src/components/sidebar/global-styles-sidebar.js

```diff
@@ -8,14 +8,28 @@ import { map } from 'lodash';
  */
 import {
 	Button,
-	__experimentalNavigation as Navigation,
```

Comment:
What's the plan regarding these components? Are they going to be refactored and use __experimentalNavigator inside?

Reply:
That's my hope yes. (I'd appreciate some help there with existing usage :P )